examples/material-ui-pigment-css-vite-ts/src/main.tsx

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import ReactDOM from "react-dom/client";
+import "@mui/material-pigment-css/styles.css";
+import App from "./App.tsx";
+
+ReactDOM.createRoot(document.getElementById("root")!).render(
+  <React.StrictMode>
```

Comment:
@siriwatknp could you please consider to add DefaultPropsProvider here?

Reply:
Sure, thanks!

Reply:
added